frontend/add-client/contact-information.component.tsx

```diff
@@ -89,14 +77,14 @@ export default function ContactInformation(props: StepComponentProps) {
         </div>
         <div>
           <label>
-            <span>State</span>
-            <StateSelect state={state} setState={setState} />
+            <span>City</span>
+            <CityInput state={state} city={city} setCity={setCity} />
```

Comment:
The reason that the city input was after the state input was so that the autocomplete for cities could autopopulate with only the cities within a certain state.

By moving the city input above it, the city input won't know which state to choose cities from if they follow the form from top to bottom.